payments-ui-core/src/test/java/com/stripe/android/ui/core/elements/CardBillingAddressElementTest.kt

```diff
@@ -57,51 +58,38 @@ internal class CardBillingAddressElementTest {
     }
 
     @Test
-    fun `Verify that when US is selected postal is not hidden`() {
-        val hiddenIdFlowValues = mutableListOf<Set<IdentifierSpec>>()
-        cardBillingElement.hiddenIdentifiers.asLiveData()
-            .observeForever {
-                hiddenIdFlowValues.add(it)
-            }
-
-        dropdownFieldController.onRawValueChange("US")
-        verifyPostalShown(hiddenIdFlowValues[0])
+    fun `Verify that when US is selected postal is not hidden`() = runTest {
+        cardBillingElement.hiddenIdentifiers.test {
+            dropdownFieldController.onRawValueChange("US")
+            verifyPostalShown(awaitItem())
+        }
     }
 
     @Test
-    fun `Verify that when GB is selected postal is not hidden`() {
-        val hiddenIdFlowValues = mutableListOf<Set<IdentifierSpec>>()
-        cardBillingElement.hiddenIdentifiers.asLiveData()
-            .observeForever {
-                hiddenIdFlowValues.add(it)
-            }
-
-        dropdownFieldController.onRawValueChange("GB")
-        verifyPostalShown(hiddenIdFlowValues[1])
+    fun `Verify that when GB is selected postal is not hidden`() = runTest {
+        cardBillingElement.hiddenIdentifiers.test {
+            skipItems(1)
```

Comment:
Why do we skip a value here and in the other tests?

Reply:
The skip is basically to just match the behavior of the tests before.

The test was only looking at the second (first index) item before, so I updated the test to only look at the second item again.